ui/panels-plugin/src/plugins/gauge-chart/GaugeChartOptionsEditor.tsx

```diff
@@ -12,12 +12,24 @@
 // limitations under the License.
 
 import { JSONEditor } from '@perses-dev/components';
-import { OptionsEditorProps } from '@perses-dev/plugin-system';
+import { OptionsEditorProps, OptionsEditorTabs } from '@perses-dev/plugin-system';
 import { GaugeChartOptions } from './gauge-chart-model';
+import { GaugeChartOptionsEditorSettings } from './GaugeChartOptionsEditorSettings';
 
 export type GaugeChartOptionsEditorProps = OptionsEditorProps<GaugeChartOptions>;
 
 export function GaugeChartOptionsEditor(props: GaugeChartOptionsEditorProps) {
   // TODO: replace with form controls for visual editing, leave temp JSON editor for thresholds
-  return <JSONEditor {...props} />;
+  return (
+    <OptionsEditorTabs
+      tabs={{
+        settings: {
+          content: <GaugeChartOptionsEditorSettings {...props} />,
+        },
+        json: {
+          content: <JSONEditor {...props} />,
+        },
```

Comment:
I think this pattern of putting the content for each tab in a separate component will help keep this parent relatively simple and also reduce merge conflicts between people working on queries vs settings.

Reply:
Really like this, I'll probably go ahead and do the same in TimeSeriesChartOptionsEditor since it's getting large